### PR TITLE
adjust status checks

### PR DESCRIPTION
**Pull request by g10**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -355,6 +355,12 @@
             "accounts_segment:update",
             "segment:update"
           ]
+        },
+        "segments": {
+          "user:update": [
+            "private_settings.lead_synchronized_segments",
+            "private_settings.contact_synchronized_segments"
```
Comment (timliuhull) [inline manifest.json]: We need to do some testing before committing this.  stephane found an edge case the other day with this logic

Comment (timliuhull) [reply manifest.json]: we can commit the other stuff without this if you'd like

**Comment by michaloo** [comment]
@g10 @timliuhull okay, it looks like I probably made a mistake here, because that status message which I was sure I saw reported as WARNING is already OK.
So actually no change needed, because that warning on duplicates probably should stay as a warning for now. I start to think about avoiding using warnings in the future, but seeing duplicates is something unhealthy but not necessarily a full-blown error, my suggestion is to keep it like this.

File: manifest.json (hunk 1 above)
Comment (g10) [reply manifest.json]: reverted this specific commit... 

🙈🙈🙈 WIP code in `develop` / `master`

**Comment by andyh565** [comment]
@michaloo should we go ahead and merge this?

**Pull request merged**